.circleci/config.yml

```diff
@@ -991,6 +1023,35 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e
 
+  test-e2e-chrome-confirmation-redesign:
+    executor: node-browsers-medium-plus
+    parallelism: 20
+    steps:
+      - run: *shallow-git-clone
+      - attach_workspace:
+          at: .
+      - run:
+          name: Move test build to dist
+          command: mv ./dist-test-confirmations ./dist
+      - run:
+          name: Move test zips to builds
+          command: mv ./builds-test-confirmations ./builds
+      - run:
+          name: test:e2e:chrome-confirmation-redesign
+          command: |
+            if .circleci/scripts/test-run-e2e.sh
+            then
+              timeout 20m yarn test:e2e:chrome --retries 2
+            fi
+          no_output_timeout: 5m
+          environment:
+            ENABLE_CONFIRMATION_REDESIGN: 1
```

Comment:
comment: we use `true` to set the value, though this should yield the same result

same comment for [line 1346](https://github.com/MetaMask/metamask-extension/pull/24240/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R1346)